<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1601592234657' stop='1601592329750'>
    <name>Paid blocks</name>
    <title>Paid blocks</title>
    <test-cases>
        <test-case start='1601592240290' status='failed' stop='1601592268032'>
            <name>MailChimp Block</name>
            <title>MailChimp Block</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1601592240291' status='undefined'>
                    <name>Can visit the block editor and add a MailChimp block</name>
                    <title>Can visit the block editor and add a MailChimp block</title>
                    <attachments>
                        <attachment title='Test failed: Paid blocks :: MailChimp Block' source='0a64be19-64e6-4147-a56f-b03281382fff-attachment.png' type='image/png' size='17194'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>TypeError: output.split is not a function</message>
                <stack-trace>TypeError: output.split is not a function
                    at split (/home/travis/build/Automattic/jetpack/tests/e2e/lib/plan-helper.js:29:16)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.&lt;computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1601592268032' status='failed' stop='1601592295738'>
            <name>Pay with PayPal</name>
            <title>Pay with PayPal</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1601592268033' status='undefined'>
                    <name>Can visit the block editor and add a Pay with PayPal block</name>
                    <title>Can visit the block editor and add a Pay with PayPal block</title>
                    <attachments>
                        <attachment title='Test failed: Paid blocks :: Pay with PayPal' source='e88c979e-69d1-4529-9b4c-6ebc37794f1b-attachment.png' type='image/png' size='17194'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>TypeError: output.split is not a function</message>
                <stack-trace>TypeError: output.split is not a function
                    at split (/home/travis/build/Automattic/jetpack/tests/e2e/lib/plan-helper.js:29:16)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.&lt;computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1601592295738' status='failed' stop='1601592323527'>
            <name>WordAds block</name>
            <title>WordAds block</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1601592295739' status='undefined'>
                    <name>Can visit the block editor and add a WordAds block</name>
                    <title>Can visit the block editor and add a WordAds block</title>
                    <attachments>
                        <attachment title='Test failed: Paid blocks :: WordAds block' source='3c3ba85e-8871-4dc3-82ad-d46ac04b87d3-attachment.png' type='image/png' size='17194'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>TypeError: output.split is not a function</message>
                <stack-trace>TypeError: output.split is not a function
                    at split (/home/travis/build/Automattic/jetpack/tests/e2e/lib/plan-helper.js:29:16)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.&lt;computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>